server/views/pages/overview/partials/_actionsCard.njk

```diff
@@ -0,0 +1,37 @@
+
+<div class="govuk-summary-card actions-card govuk-!-display-none-print" data-qa="actions-card">
+  <div class="govuk-summary-card__title-wrapper govuk-!-padding-bottom-2">
+    <h2 class="govuk-summary-card__title govuk-!-font-size-24">Actions</h2>
+  </div>
+
+  {# Review based actions #}
+  {% set showReviewActions = not induction.problemRetrievingData and induction.isPostInduction and not actionPlanReview.problemRetrievingData and reviewJourneyEnabledForPrison%}
```

Comment:
I've added the basic Reviews section 
I _think_ this is the correct IF logic; it should just work 🤞